docs/contribute/changelog.md

```
@@ -363,6 +363,17 @@ Ideally this task will be automated such that it's performed by a bot or GitHub
 If you run it from the command line, you must precede any special characters (such as backquotes) with a backslash escape character (`\`).
 :::
 
+#### CI two-step flow
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)